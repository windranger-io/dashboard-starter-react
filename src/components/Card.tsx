import { ReactNode } from 'react'

interface CardProps {
  children?: ReactNode
}

/*
 * More functionality can be added to the Card component. These should be pretty versatile
 *
 */
const Card: React.FC<CardProps> = ({ children }): JSX.Element => {
  return (
    <div className="bg-gray-50 overflow-hidden dark:bg-gray-950 sm:rounded-lg">
      <p className="hidden">simple test sample!</p>
      {children}
    </div>
  )
}

export default Card
