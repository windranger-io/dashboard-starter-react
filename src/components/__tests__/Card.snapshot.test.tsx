import React from 'react'
import renderer from 'react-test-renderer'
import Card from '../Card'

it('renders card unchanged', () => {
  const tree = renderer.create(<Card />).toJSON()
  expect(tree).toMatchSnapshot()
})
