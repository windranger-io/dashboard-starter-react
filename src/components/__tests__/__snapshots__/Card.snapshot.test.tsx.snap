// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders card unchanged 1`] = `
<div
  className="bg-gray-50 overflow-hidden dark:bg-gray-950 sm:rounded-lg"
>
  <p
    className="hidden"
  >
    simple test sample!
  </p>
</div>
`;
